config/initializers/omniauth.rb

```diff
@@ -0,0 +1,3 @@
+Rails.application.config.middleware.use OmniAuth::Builder do
+  provider :google_oauth2, ENV['GOOGLE_CLIENT_ID'], ENV['GOOGLE_CLIENT_SECRET']
```

Comment:
use `rails.application.secrets` instead of `ENV` directly.

Reply:
Done